### PR TITLE
Firefox fix

### PR DESCRIPTION
**Pull request by szczepanbarszczowski**

### CODE REVIEW

File: src/routes/Home/components/Carousel/Carousel.module.scss [hunk 1]
```diff
@@ -9,5 +9,10 @@
 
 .image {
   width: 100%;
+  -moz-user-select: none;
```
Comment (ArekRado) [inline src/routes/Home/components/Carousel/Carousel.module.scss]: we are using auto prefixer, try to add `user-select: none`

File: src/routes/Home/components/Carousel/Carousel.js [hunk 2]
```diff
@@ -28,6 +28,7 @@ export class Carousel extends PureComponent {
   state = {
     height: 480,
     wasDragged: false,
+    mousePositon: {},
```
Comment (ArekRado) [inline src/routes/Home/components/Carousel/Carousel.js]: add x and y (no-empty-objects :D)

File: src/routes/Home/components/Carousel/Carousel.module.scss (hunk 1 above)
Comment (szczepanbarszczowski) [reply src/routes/Home/components/Carousel/Carousel.module.scss]: Will work only for user-select. user-drag was removed from css specification but still working on browsers so we need to add prefixes for webkit and moz browsers

File: src/routes/Home/components/Carousel/Carousel.js (hunk 2 above)
Comment (szczepanbarszczowski) [reply src/routes/Home/components/Carousel/Carousel.js]: +1

**Pull request merged**